spec/imsc-hrm.html

```diff
@@ -425,8 +425,8 @@ <h2>Architecture</h2>
       <a>Intermediate Synchronic Documents</a> E<sub>i</sub> obtained from an input <a>IMSC Document Instance</a>:</p>
       <ul>
         <li><a>non-empty ISDs</a> are processed using a simple double buffering model: while a <a>non-empty ISD</a>
-        E<sub>n</sub> is being painted into a Back Buffer, the previous <a>non-empty ISD</a> E<sub>m</sub> is available
-        for display in a Front Buffer; and</li>
+        E<sub>n</sub> is being painted into a Back Buffer by the Presentation Compositor, the previous <a>non-empty
```

Comment:
Seems like we should normatively define "Presentation Compositor"?

Reply:
Do you mean with a `dfn`?

Reply:
Yes

Reply:
Definitions are pretty inconsistent in the document and we already a [separate issue](https://github.com/w3c/imsc-hrm/issues/47) to address that.